packages/react-menu/README.md

```diff
@@ -0,0 +1,11 @@
+# @fluentui/react-menu
+
+**React Menu components for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+
+These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+
+To import React Menu components:
+
+```js
+import { ComponentName } from '@fluentui/react-menu';
+```
```

Comment:
```suggestion
```

Let's remove for now

Reply:
Yeah makes sense if we have to publish it